test/fileurlreader.cpp

```diff
@@ -114,15 +114,9 @@ TEST_CASE("URL reader returns error structure if file cannot be opened",
 	SECTION("reload() returns error structure if file does not exist") {
 		const auto error_message = u.reload();
 		REQUIRE(error_message.has_value());
-		/**
-		SECTION("the error message contains the filename") {
-			INFO("error_message: " + error_message.value().message);
-			REQUIRE(error_message.value().message.find(urlsFile.get_path()) != std::string::npos);
-		}
-		*/
 		SECTION("the error structure contains error message and kind of an error") {
 			INFO("error_message: " + error_message.value().message);
-			REQUIRE(error_message.value().kind == utils::ReadTextFileErrorKind::CantOpen);
+			REQUIRE(!error_message.value().message.empty());
```

Comment:
```suggestion
			REQUIRE_FALSE(error_message.value().message.empty());
```

This catches more attention and is thus more idiomatic for this particular test framework.